test/spec.js

```diff
@@ -2,6 +2,9 @@ import 'should';
 import wiki from '../src/wiki';
 import nock from 'nock';
 import fs from 'fs';
+// import rewire from 'rewire';
+
+const described = () => {};
```

Comment:
Used it to disable certain tests while I was writing new cases (change `describe` to `described` on ones I didn't want to run). Total kludge and forgot to remove it when I was done :) There's probably a better way to disable them that I'm not aware of.